### ISSUE TITLE
PESQ is smaller than paper

### ISSUE THREAD

**Issue by LexiYIN** [facebookresearch/denoiser#72]
Hi~
  I train the model on the Valentini's dataset with default setting(stft_loss is True) and not use the pre-trained, but the test data's PESQ is about 2.679, any pre-trained model only on the Valentini's dataset can get the best result fit the paper? 
Thanks.

**Comment by adefossez** [comment]
Hey @LexiYIN 
Do you have the training log ? would it be possible to share it here ? You can find it in `outputs/`, it should be one of the folder  with `dset=valentini`. Also you might want to check which subjects you used for the validation set, we used speakers p286 and p287.

**Comment by aloha0920** [comment]
Hi @adefossez ,
do you use 28spk as training data and use p268 p287 as validation set right?

**Comment by adefossez** [comment]
Speaker **286** and 287 for validation right. And the rest (so only 26 speakers) as the train set :)
